ipaserver/install/dsinstance.py

```diff
@@ -544,6 +544,8 @@ def __create_instance(self):
         slapd_options = Slapd2Base(logger)
         slapd_options.set('instance_name', self.serverid)
         slapd_options.set('root_password', self.dm_password)
+        # don't create SSCA and temporary self-signed certificate
+        slapd_options.set('self_sign_cert', False)
```

Comment:
It's fine I guess but do we need a comment here? Is the option self-explanatory enough?